xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/historyinline.vm

```diff
@@ -19,7 +19,7 @@
 #set ($discard = $collectionstool.reverse($versions))
 #set ($totalVersions = $versions.size())
 #if ($totalVersions == 0)
-  #warning ($services.localization.render('thispagedoesnotexist'))
+  #warning ($services.localization.render(core.history.empty''))
```

Comment:
You sure about that ? :)

Reply:
You are right, I should have used `core.viewers.history.*` instead ;).